src/Microsoft.ML.Data/Transforms/ChooseColumnsTransform.cs

```diff
@@ -58,6 +58,20 @@ public bool TryUnparse(StringBuilder sb)
 
         public sealed class Arguments
         {
+            public Arguments()
+            {
+
+            }
+
+            public Arguments(params string[] columns)
```

Comment:
>public [](start = 12, length = 6)

should it be private/internal?

Reply:
It could be if you don't foresee anyone else interested in using it?

Reply:
Just as a higher level point, we always have a tendency to make things as private as they reasonably can be, both because it makes the code much easier to understand, and also we will eventually have the problem that something "unhidden" can never be hidden again.

---
In reply to: [201022055](https://github.com/dotnet/machinelearning/pull/491#discussion_r201022055) [](ancestors = 201022055,200723744)